application/Modules/CustomFields/Models/CustomField.php

```
@@ -38,27 +42,45 @@ public static function get_nicename($element)
+     * Legacy migration info:
+     * @legacy-file application/modules/custom_fields/models/Mdl_custom_field.php
+     * @legacy-function custom_types()
      */
     public static function custom_types()
     {
         $CI = &get_instance();
-        $CI->load->model('custom_values/mdl_custom_values');
+        $CI->load->model('custom_values/custom_value');
 
         return Mdl_Custom_Values::custom_types();
     }
```

PHPMD (2.15.0)

The method custom_types is not named in camelCase. (undefined)

(CamelCaseMethodName)